### ISSUE TITLE
bug in re-zero dialog box before starting a job?

### ISSUE THREAD

**Issue by secretspy711** [zapmaker/GrblHoming#85]
Generally I raise my bit up 10mm before starting a job, which allows me to turn on the spindle without marking the surface.  When I did this and then hit "begin," grbl controller popped up a dialog box asking me if I wanted to first zero at this location before starting.  I clicked "no," since I had already done that when the bit was touching the material.  It proceeded to go to the first cutting position but did not drop back down by the 10mm before cutting (causing it to cut in air).  I tried again, this time clicking "yes" on the popup, with the same result.  I think this is a bug?
